ios/start.md

```diff
@@ -53,10 +52,9 @@ Create new AWS backend resources and pull the AWS services configuration into th
 ```bash
 $ cd ./YOUR_PROJECT_FOLDER
 $ amplify init        #accept defaults
-$ amplify push        #creates configuration file
```

Comment:
Push is needed to kick off a codegen

Reply:
seems like on amplify cli version 1.7.1, amplify push has no changes, and amplify init already created the configuration file

Reply:
Config file is created yes but we need a push to do the codegen of GraphQL types from the SDL. It's also the flow we want to ensure a path to success for creation of the backend for someone getting started.

Reply:
in Step 4: Add API and Database, yes, ie.
```
amplify add api
amplify push
```
but this one is just to create configuration file, and choose the AWS account, etc. with single `amplify init` command. when I tried to amplify push, there was nothing pending to be pushed

Reply:
Ok it looks like we had a CLI behavior change, this was not the case before.

Reply:
We should validate for Android as well

Reply:
good idea, will report back